app/models/ability.rb

```diff
@@ -29,6 +29,10 @@ def initialize(user)
       user.admin? or (user.student? && (team.students.include? user))
     end
 
+    can :see_offered_conferences, Team do |team|
+      user.admin? or (team.students.include? user) or team.supervisors.include?(user)
```

Comment:
Super minor thing, but it's considered bad practice to use `or` or `and` as they may introduce bugs and do only add very little readability. It would also be nice if you could align the usage of brackets here, would be easier to grasp I think:
```ruby
user.admin? || team.students.include?(user) || team.supervisors.include?(user)
```